### PR TITLE
Billy operator

### PR DESCRIPTION
**Pull request by billwpierce**
Added a new operator class for my preferences [    :)     ]. Adds buttons 7, 8, and 10 to be able to optionally control BallIO with a free hand.

### CODE REVIEW

File: src/org/usfirst/frc4904/robot/humaninterface/operators/BillyOP.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import org.usfirst.frc4904.robot.commands.GearioIntake;
+import org.usfirst.frc4904.robot.commands.GearioOuttake;
+import org.usfirst.frc4904.robot.commands.SetOverride;
+import org.usfirst.frc4904.robot.commands.SetRampState;
+import org.usfirst.frc4904.robot.commands.Shoot;
+import org.usfirst.frc4904.robot.subsystems.GearIO;
+import org.usfirst.frc4904.standard.humaninput.Operator;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class BillyOP extends Operator {
```
Comment (c0d3rman) [inline src/org/usfirst/frc4904/robot/humaninterface/operators/BillyOP.java]: I would personally go with `Billy` or `BillyOperator`.

File: src/org/usfirst/frc4904/robot/humaninterface/operators/BillyOP.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+	public static final double INTAKE_THRESHOLD = 0.5;
+
+	public BillyOP() {
+		super("DefaultOperator");
```
Comment (c0d3rman) [inline src/org/usfirst/frc4904/robot/humaninterface/operators/BillyOP.java]: Make sure to change this to the name of your class so it displays properly on SmartDashboard!

File: src/org/usfirst/frc4904/robot/humaninterface/operators/BillyOP.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+				.start();
+		RobotMap.Component.gearIO.setRampState(GearIO.RampState.EXTENDED);
+		RobotMap.Component.teensyStick.getButton(11).whenPressed(new SetOverride(true, RobotMap.Component.gearIO));
+		RobotMap.Component.teensyStick.getButton(11).whenReleased(new SetOverride(false, RobotMap.Component.gearIO));
+		RobotMap.Component.teensyStick.getButton(12)
+			.whenPressed(new SetRampState(GearIO.RampState.EXTENDED));
+		RobotMap.Component.teensyStick.getButton(13).whenPressed(new SetRampState(GearIO.RampState.RETRACTED));
+	}
+
+	private class ThresholdCommand extends Command {
```
Comment (c0d3rman) [inline src/org/usfirst/frc4904/robot/humaninterface/operators/BillyOP.java]: Now that we actually have more than one operator we should probably move this to `standard` (not necessarily as part of this PR though).

**Comment by billwpierce** [comment]
@c0d3rman I made the changes you requested. I do agree about putting threshold command in standard, but I think for now if this branch is good to go, give it a 👍 and I can test it next time I have operator practice (which is hopefully tomorrow), and then we can get it merged.

**Review by c0d3rman — APPROVED** [review]

**Comment by c0d3rman** [comment]
Huh - your builds are failing. Your branch seems to be referencing a nonexistent version of standard.

**Comment by billwpierce** [comment]
@c0d3rman ready to merge

**Pull request merged**